packages/protocol/contracts/common/Accounts.sol

```diff
@@ -83,6 +113,26 @@ contract Accounts is
   function initialize(address registryAddress) external initializer {
     _transferOwnership(msg.sender);
     setRegistry(registryAddress);
+    setEip712DomainSeparator();
```

Comment:
this won't be called as this contract can't be reinitialized (without a storage upgrade)